tests/internal/client/client.go

```
@@ -0,0 +1,198 @@
+			return got
+		}
+	}
+	c.t.Fatalf("client: timed out waiting for message with prefix %q", prefix)
+	return ""
+}
+
+// ExpectNoMessage asserts the server sends nothing within the given window.
+// A received message causes the test to fail.
+func (c *Client) ExpectNoMessage(window time.Duration) {
+	c.t.Helper()
+	c.conn.SetReadDeadline(time.Now().Add(window))
```

golangci-lint (2.12.2)

[error] Error return value of `c.conn.SetReadDeadline` is not checked

(errcheck)